hieradata/common.yaml

```diff
@@ -413,7 +413,9 @@ govuk::apps::release::github_username: "govuk-ci"
 govuk::apps::rummager::nagios_memory_warning: 4600
 govuk::apps::rummager::nagios_memory_critical: 4900
 govuk::apps::rummager::enable_publishing_listener: true
+govuk::apps::rummager::enable_govuk_index_worker: true
```

Comment:
If this is still in the testing phase would it be better to only enable this in integration?